docs/en/ingest-management/create-agent-policies-no-UI.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[create-a-policy-no-ui]]
+= Create an agent policy without using the UI
+
+For use cases where you want to provide a default agent policy or support
+automation, you can set up an agent policy without using the {fleet} UI. To do
+this, either use the {fleet} API or add a preconfigured policy to {kib}:
+
+[discrete]
+[[use-api-to-create-policy]]
+== Option 1. Create an agent policy with the API
+
+[source,sh]
+----
+POST <kibana_host:port>/api/fleet/agent_policies?sys_monitoring=true
```

Comment:
@juliaElastic For this example, I'm wondering if it might be better to show a curl command since the Kibana APIs can't (AFAIK) be run from the dev console:

```
curl -u <username>:<password> --request POST \
  --url <kibana_url>/api/fleet/agent_policies?sys_monitoring=true \
  --header 'content-type: application/json' \
  --header 'kbn-xsrf: true' \
  --data '{"name":"Agent Policy 1","namespace":"default","monitoring_enabled":["logs","metrics"]}'
```

WDYT?

Reply:
curl is good, thanks